src/dynamic-checkout/config/assets.ts

```diff
@@ -2,8 +2,20 @@ module ProcessOut {
   export const CARD_SCHEMES_ASSETS = {
     visa: "/images/schemes/rectangle/visa.svg",
     mastercard: "/images/schemes/rectangle/mastercard.svg",
-    amex: "/images/schemes/rectangle/americanexpress.svg",
```

Comment:
sanity check: `amex` and `unionpay` are fully moved to `american-express` and `union-pay` now?